include/ranges.h

```diff
@@ -84,17 +430,15 @@ struct chunk {
 
 template <int Dims>
 struct subrange {
-	static_assert(Dims > 0);
-
-	static constexpr int dims = Dims;
+	static constexpr int dimensions = Dims;
 
 	celerity::id<Dims> offset;
-	celerity::range<Dims> range = detail::range_cast<Dims>(celerity::range<3>(0, 0, 0));
+	celerity::range<Dims> range = detail::zero_range;
 
 	subrange() = default;
 
 	// Due to an apparent bug in the ComputeCpp 2.6.0 compiler, the implicit copy constructor of subrange instantiates a copy constructor of the "array" type
-	// underlying ComputeCpp's implementation of sycl::range. This generated array copy constructor receives a *non-const* lvalue instead of a const lvalue,
+	// underlying ComputeCpp's implementation of celerity::range. This generated array copy constructor receives a *non-const* lvalue instead of a const lvalue,
```

Comment:
Wrong auto-replacement; also this workaround can presumably be removed now!

Reply:
Removed.